app/components/post_list/post/index.ts

```diff
@@ -145,11 +145,20 @@ const withPost = withObservables(
             distinctUntilChanged(),
         );
 
+        const hasFiles = post.files.observe().pipe(
+            switchMap((ff) => of$(Boolean(ff.length))),
+            distinctUntilChanged(),
+        );
+
+        const hasReactions = post.reactions.observe().pipe(
+            switchMap((rr) => of$(Boolean(rr.length))),
+            distinctUntilChanged(),
+        );
+
```

Comment:
We don't need really the counts for these, just whether they have more than one element, so I got the chance to remove the `observeCount` from here, to make the flipper results more readable.